includes/widgets/playing-cards.php

```diff
@@ -0,0 +1,500 @@
+				'options' => [
+					'default' => esc_html__( 'Default Color', 'elementor' ),
+					'custom' => esc_html__( 'Custom', 'elementor' ),
+				],
+			]
+		);
+
+		$this->add_responsive_control(
+			'card_size',
+			[
+				'label' => esc_html__( 'Cards Suite Size', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'size_units' => [ 'px', 'rem', 'vw', 'custom' ],
+				'range' => [
+					'px' => [
+						'min' => 20,
+						'max' => 80,
+					],
+				],
+				'selectors' => [
+					'{{WRAPPER}} {{CURRENT_ITEM}}.e-card-suite' => '--e-card-suite-size: {{SIZE}}{{UNIT}}',
+				],
+			]
+		);
+
+		$this->add_control(
+			'card_padding',
+			[
+				'label' => esc_html__( 'Cards Padding', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'size_units' => [ 'px', 'rem', 'vw', 'custom' ],
+				'range' => [
+					'px' => [
+						'min' => 0,
+						'max' => 20,
+					],
+				],
+				'selectors' => [
+					'{{WRAPPER}} {{CURRENT_ITEM}}.e-playing-card' => '--e-card-padding: {{SIZE}}{{UNIT}}',
+				],
+			],
+		);
+
+		$this->end_controls_section();
+	}
+
+
+	/**
+	 * Render Playing cards widget output on the frontend.
+	 *
+	 * Refactored to use helper methods for rendering controls and individual cards.
+	 *
+	 * @since 1.0.0
+	 * @access protected
+	 */
+	protected function render(): void {
+		$settings = $this->get_settings_for_display();
+
+		if ( empty( $settings['cards'] ) ) {
+			return;
+		}
+
+		$this->render_card_controls();
+
+		echo '<div class="e-playing-cards-container e-flex card-section">';
+		foreach ( $settings['cards'] as $index => $card ) {
+			$this->render_card( $index, $card );
+		}
+		echo '</div>';
+
+		echo '<div class="e-playing-cards-container-clone e-flex card-section">';
+		echo '</div>';
+	}
+
+	/**
+	 * Render card controls.
+	 *
+	 * @since 1.0.0
+	 * @access protected
+	 */
+	protected function render_card_controls(): void {
+		?>
+		<div class="playing-cards-controls e-button-container mb-2">
+			<button type="button" class="e-button cards-play">
+				<?php esc_html_e( 'Play', 'elementor' ); ?>
+			</button>
+			<button type="button" class="e-button cards-shuffle">
+				<?php esc_html_e( 'Shuffle', 'elementor' ); ?>
+			</button>
+			<button type="button" class="e-button cards-play-again">
+				<?php esc_html_e( 'Play Again', 'elementor' ); ?>
+			</button>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Render an individual card.
+	 *
+	 * @param int $index Index of the card.
+	 * @param array $card Card data.
+	 * @since 1.0.0
+	 * @access protected
+	 */
+	protected function render_card( int $index, array $card ): void {
+		$link_key = 'link_' . $index;
+
+		$this->add_render_attribute($link_key, 'class', [
+			'e-playing-card',
+			'elementor-repeater-item-' . esc_attr( $card['_id'] ),
+			'playing-card-bg',
+		]);
+		?>
+		<div <?php $this->print_render_attribute_string( $link_key ); ?> data-value="<?php echo esc_attr( $card['_id'] ); ?>">
+			<div class="e-card-inner">
+				<div class="e-card-inner-back">
+					⊖
```

Comment:
What is this?

Reply:
Do we need this symbol? `⊖`
It is fine. I was just wondering what is was.
Maybe you could handle it with CSS as well.

Reply:
if we can do something via HTML why we want to use CSS?

Reply:
It is not so important here.